ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -427,7 +427,32 @@ aws|aws-arm64|aws-usgov)
     ;;
 esac
 
-sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
```

Comment:
I dunno why folks have copy/pasted this around so much, but:

```console
$ git --no-pager grep cvo-overrides.yaml ci-operator/step-registry/
ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh:sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh:sed -i '/^  channel:/d' ${dir}/manifests/cvo-overrides.yaml
ci-operator/step-registry/ipi/install/powervs/install/ipi-install-powervs-install-commands.sh:sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh:sed -i '/^  channel:/d' "manifests/cvo-overrides.yaml"
ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh:sed -i '/^  channel:/d' "manifests/cvo-overrides.yaml"
ci-operator/step-registry/upi/conf/vsphere/zones/upi-conf-vsphere-zones-commands.sh:sed -i '/^  channel:/d' "manifests/cvo-overrides.yaml"
ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:sed -i '/^  channel:/d' ${ARTIFACT_DIR}/installer/manifests/cvo-overrides.yaml
ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh:sed -i '/^  channel:/d' manifests/cvo-overrides.yaml
```

I'm completely fine leaving those other steps alone, and letting their maintainers continue to copy/paste or find some new way to DRY things up, but wanted to point out the issue in case other folks wanted to do something more proactive.

Reply:
I think I will try to update them all, but I will do that in separate PRs otherwise the rehearsals will be pretty hard to track.